core/tests/style/pack/test_apply.py

```diff
@@ -59,17 +59,20 @@ def test_set_background_color():
     root._impl.set_background_color.assert_called_once_with(rgb(255, 255, 255))
 
 
-def test_set_font():
-    root = ExampleNode(
-        "app",
-        style=Pack(
+@pytest.mark.parametrize("use_shorthand", [True, False])
```

Comment:
As a stylistic thing - my preference here would be to factor out the arguments to `Pack` as the parameterised argument - that way, we're testing multiple ways to construct the same thing, rather than two independent constructors that happen to have the same result.